profile/faq.md

```diff
@@ -0,0 +1,82 @@
+# Frequently Asked Questions
+
+: Last Updated: 8 June 2025
+
+# General Questions
+
+## How do I use this?
+
+Our MVP product is a WordPress plugin, the FAIR Plugin. You can install it just like you would any other plugin.
+
+You can download the latest version from the [FAIR Plugin releases page](https://github.com/fairpm/fair-plugin/releases).
+
+## Does it matter to users where or how they get their WordPress downloads?
+
+It's true that many users do not actively think about where they get WordPress. As caretakers of a significant part of the web, we aim to provide secure and reliable access for them so they don't have to care, while helping developers ensure the right code rolls out. Federating the WordPress ecosystem adds redundancy and resilience.
+
+## Will you do X, Y,or Z?
+
+We're always happy for more community members and outside voices to suggest future paths for FAIR. If you have an idea you'd like to share, post in our [ideas forum][https://github.com/orgs/fairpm/discussions/categories/ideas].
+
+For more information on our process, read up on our [Technical Steering Committee](https://github.com/fairpm/tsc).
+
+# Security and Reliability
+
+We take security very seriously and believe that using FAIR will fortify the WordPress supply chain. Using signing and authentication, FAIR mitigates unapproved takeovers or nulling of plugins or themes, restoring trust in deliverables. 
+
+One of the main reasons the FAIR project exists is to ensure that all WordPress users, regardless of their host, continue to have access to WordPress, theme and plugin updates. This will give developers and users confidence in the security of their software.
+
+## Could FAIR increase the risk of a DDoS attack on WordPress.org?
+
+No. The plugin and theme repository mirroring part of FAIR currently uses AspirePress, which caches plugins and themes, significantly reducing traffic to WordPress.org. Their mirror has been running without issue since the middle of January 2025. Other APIs, such as the events listing or BrowseHappy, fully replace the official ones. This effectively reduces load rather than increasing it, as many APIs are no longer sending any traffic to WordPress.org at all.
+
+## If you use AspirePress, how does AspirePress handle load to avoid overwhelming WordPress.org?
+
+AspireSync, used by AspirePress, employs strict rate-limiting (no more than 10 simultaneous requests) and pauses to minimize impact. WordPress.org also has its own traffic shaping measures to manage potential overloads.
```

Comment:
perhaps "AspireSync, used by AspirePress to fetch WordPress.org data,"?

I think it's beneficial to describe the service referenced briefly, to give that more context.